_episodes/09-pull-requests.md

```diff
@@ -192,99 +291,224 @@ Largest City: Paris
 {: .output}
 
 Next we will add and commit these changes.
-Then we will push them to our forked version of the repo.
+Then we can push them to our forked version of the repo.
 
 ~~~
 $ git add france.txt
 $ git commit -m "Added largest city to france file"
-$ git push origin master
+$ git push origin addFrance
 ~~~
 {: .bash}
 
+~~~
+Counting objects: 3, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (3/3), done.
+Writing objects: 100% (3/3), 387 bytes | 0 bytes/s, done.
+Total 3 (delta 0), reused 0 (delta 0)
+To https://github.com/sstevens2/countries.git
+   31aa2e3..609acfe  addFrance -> addFrance
+~~~
+{: .output}
+
 Now if we reload the pull request.  The new commit was added to that pull request.
+New commits pushed to the same branch are included in the previous pull request.
+If you want to suggest changes separately you need to make separate branches but 
+if you want the changes to be considered together you should put them in the same branch.
+
+![](../fig/github_screenshot_after_new_commit.png)
+
+When working with others we might encounter the conflicts, which we
+learned about earlier in branches.  Let's practice resolving conflicts when working
+collaboratively.
+
+We will continue to work in the `addFrance` branch from before and check we are
+in that branch before we start.
+
+
+~~~
+$ git branch
+~~~
+{: .bash}
+
+~~~
+* addFrance
+  master
+~~~
+{: .output}
+
+
+Next we will each add our country to the existing `README.md` file in the repository in the line directly following the `Countries:` line.
+
+~~~
+$ nano README.md
+$ cat README.md
+~~~
+{: .bash}
+
+~~~
+# countries
+Sandbox for learning PR's in Software Carpentry workshop
+
+Countries:
+France
+~~~
+{: .output}
+
+Next we need to add, commit, and push these requests to our existing pull request.
+
+~~~
+$ git add README.md
+$ git commit -m "Added France to list of countries in README"
+$ git push origin addFrance
+~~~
+{: .bash}
+
+~~~
+Counting objects: 3, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (3/3), done.
+Writing objects: 100% (3/3), 376 bytes | 0 bytes/s, done.
+Total 3 (delta 1), reused 0 (delta 0)
+remote: Resolving deltas: 100% (1/1), completed with 1 local object.
+To https://github.com/sstevens2/countries.git
+   609acfe..66d7ebf  addFrance -> addFrance
+~~~
+{: .output}
+
+Now if we reload the page we had a pull request we notice that our `addFrance`
+branch is conflicting with upstream's `master` branch.
+This is because someone else edited the same line of the `README.md` file by 
+adding 'United States' where we added 'France'.
+
+![](../fig/github_screenshot_conflicting_PR.png)
+
+It is possible to resolve this conflict in github by clicking the 'Resolve Conflicts' button.
```

Comment:
I had never done it in github so I actually tested out resolving it in github and I liked it better locally but this might just be my preference.  I was considering adding that info in but think it might just be a personal opinion.